Library/ViewModels/ChangeEmailViewModelTests.swift

```diff
@@ -133,7 +133,7 @@ final class ChangeEmailViewModelTests: TestCase {
 
       // Disabled if new email is equal to the old one.
       self.vm.inputs.emailFieldTextDidChange(text: "ksr@kickstarter.com")
-      self.saveButtonIsEnabled.assertValues([true, false])
+      self.saveButtonIsEnabled.assertValues([true, true])
```

Comment:
I tried changing the template email to the "kst@kickstarter.com" but it still sends true back as its last value.

Reply:
So not sure.

Reply:
In app functionality only enables the "Save" button when the email is different..

Reply:
Yeah I'm under the impression this was a bug that was never addressed in the test because in this particular unit test the email address is changing therefore the save button should be enabled.